numpy/_core/src/umath/loops_unary_fp.dispatch.cpp

```diff
@@ -0,0 +1,352 @@
+#define _UMATHMODULE
+#define _MULTIARRAYMODULE
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+#include "numpy/ndarraytypes.h"
+#include "numpy/npy_common.h"
+#include "numpy/npy_math.h"
+#include "numpy/utils.h"
+#include "fast_loop_macros.h"
+#include "loops_utils.h"
+#include <hwy/highway.h>
+#include <hwy/aligned_allocator.h>
+
+
+namespace hn = hwy::HWY_NAMESPACE;
+HWY_BEFORE_NAMESPACE();
+template <typename T>
+struct OpRound {
+  HWY_INLINE hn::VFromD<hn::ScalableTag<T>> operator()(
+      hn::VFromD<hn::ScalableTag<T>> a) {
+#if HWY_TARGET >= HWY_SSSE3
```

Comment:
We had a discussion around this in the optimisation team meeting just now, this logic needs to be limited to `x86` as it references x86 features:
```suggestion
#if HWY_ARCH_X86_64 && HWY_TARGET >= HWY_SSSE3
```

At the moment, from what I understood, `HWY_TARGET` is a shared bit-field and this targets everything non-x86 as well as x86 better than `SSSE3`.